src/EditorFeatures/Test2/IntelliSense/CSharpCompletionCommandHandlerTests.vb

```diff
@@ -588,7 +588,7 @@ class C
 }]]></Document>)
 
                 state.SendTypeChars("i")
-                Await state.AssertSelectedCompletionItem(displayText:="int", isHardSelected:=True)
+                Await state.AssertSelectedCompletionItem(displayText:="in", isHardSelected:=True)
```

Comment:
If this is because it's a lambda now, then we change this test to be tying "float" or something else. This has removed the coverage for the intended scenario.